tox.ini

```diff
@@ -0,0 +1,13 @@
+[tox]
+envlist = py27
+
+[testenv]
+usedevelop = true
+deps =
+    -rrequirements.txt
+    pyftpdlib
+    pytest-xdist
+commands =
+    py.test {posargs:tests}
```

Comment:
I do not like `python setup.py test`.

The very-long-term direction in the Python packaging ecosystem is to get rid of `setup.py`, see e.g. [PEP-517](https://www.python.org/dev/peps/pep-0517/).   It will take many years, but in the mean time I don't think it's worth to spend effort adding features in a build system that's going to be eventually deprecated.